x-pack/solutions/search/plugins/search_synonyms/public/plugin.ts

```diff
@@ -30,7 +31,7 @@ export class SearchSynonymsPlugin
     }
     core.application.register({
       id: PLUGIN_ID,
-      appRoute: '/app/elasticsearch/synonyms',
+      appRoute: PLUGIN_ROUTE_ROOT,
       title: PLUGIN_TITLE,
       deepLinks: [
         {
```

Comment:
Not related to this PR but this deep link definition seems redundant. the app will automatically get a deep link registered for the `PLUGIN_ID` at the `appRoute` and it will have the same title. we should be able to add `globalSearch` to the register object's `visibleIn` instead of adding a 2nd deeplink to the same path. Unless I'm missing something.

Reply:
I will try removing.

Reply:
Without the deeplinks, the path is not visible in the Stack. I will revert it back until we find a different solution.

Reply:
What do you mean by?

> the path is not visible in the Stack